From 115f88a3e7901541a67964d41bbd2a1866cfc521 Mon Sep 17 00:00:00 2001
From: Ken Dreyer <ktdreyer@ktdreyer.com>
Date: Thu, 8 Aug 2013 18:21:15 -0600
Subject: [PATCH] more texinfo 5.1 hacks

For hx509.texi, we need the copyright macros for both html and info.
Just remove the "ifhtml" conditionals.

For whatis.texi, texinfo 5.1 inserts the sub{} macro inline with no
newlines, so there are errors about @html not being at the end of a
line, etc.
---
 doc/hx509.texi  | 2 --
 doc/whatis.texi | 2 ++
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/hx509.texi b/doc/hx509.texi
index b44cbea..d050c21 100644
--- a/doc/hx509.texi
+++ b/doc/hx509.texi
@@ -39,14 +39,12 @@
 @def@copyrightstart{}
 @def@copyrightend{}
 @end iftex
-@ifinfo
 @macro copynext
 @end macro
 @macro copyrightstart
 @end macro
 @macro copyrightend
 @end macro
-@end ifinfo
 
 @page
 @copyrightstart
diff --git a/doc/whatis.texi b/doc/whatis.texi
index 891e275..7d83725 100644
--- a/doc/whatis.texi
+++ b/doc/whatis.texi
@@ -43,9 +43,11 @@ services can authenticate each other.
 
 @ifhtml
 @macro sub{arg}
+
 @html
 <sub>\arg\</sub>
 @end html
+
 @end macro
 @end ifhtml
 
